json-rpc/Sources/JsonRpc/Client.swift

```diff
@@ -198,6 +211,8 @@ private class Handler: ChannelInboundHandler, ChannelOutboundHandler {
         switch error {
         case CodecError.requestTooLarge, CodecError.badFraming, CodecError.badJson:
             promise.succeed(JSONResponse(id: requestId, errorCode: .parseError, error: error))
+        case is NIOExtras.NIOExtrasErrors.LeftOverBytesError:
+            promise.succeed(JSONResponse(id: requestId, errorCode: .parseError, error: error))
```

Comment:
should it be merged into the previous case?

Reply:
aye